go-controller/pkg/cluster/master.go

```diff
@@ -25,6 +186,11 @@ import (
 // TODO: Verify that the cluster was not already called with a different global subnet
 //  If true, then either quit or perform a complete reconfiguration of the cluster (recreate switches/routers with new subnet values)
 func (cluster *OvnClusterController) StartClusterMaster(masterNodeName string) error {
+	err := cluster.checkOvnDBExist(masterNodeName)
```

Comment:
I think checkOvnDBExist should be renamed to something like checkOvnDBIsMaster

Reply:
Here we are not checking cluster master ls anymore. We just check default namespace annotation and if annotated IP changed for db existence.